app/src/main/java/org/ole/planet/myplanet/utils/RealmUtils.kt

```
@@ -0,0 +1,47 @@
+        try {
+            val cacheKey = Pair(obj.javaClass, fieldName)
+            var entry = fieldCache[cacheKey]
+
+            if (entry == null) {
+                var clazz: Class<*>? = obj.javaClass
+                var field: Field? = null
+
+                while (clazz != null && field == null) {
+                    try {
+                        field = clazz.getDeclaredField(fieldName)
+                    } catch (e: NoSuchFieldException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)